tests/test_quickstart.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
Let's rename this file `test_coiled.py` as we'll want to add more related tests here shortly